packages/contentful/migrations/crn/researchOutputs/20230724090200-unique-urls.js

```diff
@@ -0,0 +1,17 @@
+module.exports.description = 'Remove uniqueness constraint on link field';
```

Comment:
```suggestion
module.exports.description = 'Add URL validation on link field';
```

Reply:
The URL validation is there before this migration. What this migration is doing is setting the validation to _only_ that, and without the uniqueness validator.